src/engraving/accessibility/accessibleroot.cpp

```diff
@@ -119,7 +119,7 @@ void AccessibleRoot::updateStaffInfo(const AccessibleItem* newAccessibleItem, co
 
     if (newAccessibleItem && newAccessibleItem->element()->hasStaff()) {
         staff_idx_t newStaffIdx = newAccessibleItem->element()->staffIdx();
-        staff_idx_t oldStaffIdx = oldAccessibleItem ? oldAccessibleItem->element()->staffIdx() : nidx;
+        staff_idx_t oldStaffIdx = oldAccessibleItem && oldAccessibleItem->registered() ? oldAccessibleItem->element()->staffIdx() : nidx;
```

Comment:
ok I put up a fix that's almost the same. But I think it's wrong - the problem is that AccessibilityRoot::m_focusedElement is often referring to an already deleted object, which we should ensure doesn't happen using some sort of smart pointer.